RELEASE.md

```diff
@@ -45,11 +45,65 @@ WIP: this document is still incomplete.
     TODO: this script should also regenerate
     * changelog
     * python api client
-1. Wait and make sure the cloudbuild job that builds all images in gcr.io/ml-pipeline-test succeeded for above commit. Then submit the second cloudbuild job that copies these images to gcr.io/ml-pipeline.
+1. View related cloudbuild jobs' statuses by clicking the latest commit's status icon
+in the release branch. The page will look like https://github.com/kubeflow/pipelines/runs/775788343.
+1. Wait and make sure the `build-each-commit` cloudbuild job that builds all images
+in gcr.io/ml-pipeline-test succeeded. If it fails, please click "View more details on Google Cloud Build" and then "Retry".
+1. Select the `release-on-tag` cloudbuild job that copies built images and artifacts to
+public image registry and gcs bucket. This job should have already failed because
+artifacts haven't been built. Now, please click "View more details on Google Cloud Build"
+and then "Retry", because after waiting for previous step, artifacts are now ready.
+
+    TODO: we should have an automation KFP cluster, and the waiting and submiting
+    `release-on-tag` cloudbuild task should be automatically waited.
+1. Refer to internal doc for getting password of kubeflow-pipelines user in PyPI.
```

Comment:
Is this the Valentine password? If so, we can be specific by saying valentine password here since we probably can't put the link of the internal doc here and instead we can give ourselves a clue by saying valentine password....

Reply:
Well, I'd suggest searching by PyPI, because it doesn't seem suitable to mention valentine here either.